### ISSUE TITLE
Zsnes 1.51 emulator is offkey

### ISSUE THREAD

**Issue by ualaa1** [joncampbell123/dosbox-x#3178]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar bug report(s) already exists?

- [X] I have searched and didn't find any similar issues.

### What operating system(s) this bug have occurred on?

Windows 10 64 bit

### What version(s) of DOSBox-X have this bug?

0.83.20

### Describe the bug

This is especially noticable in these videos on the title screen. Aplogies for the lags on both of these videos, I don't have a good Pc

On Dosbox-X: https://imgur.com/a/32Kljx8

On a different DOS Emulator (correct pitch, earrape and freeze at the end due to possible bugs on that emulator or the fact that its only emulating 133mhz instead of the correct 300+): https://user-images.githubusercontent.com/68984176/147443845-f9a0a37c-ef87-4f35-82ad-2acad191dc4d.mp4

What it sound like on a real SNES: https://www.youtube.com/watch?v=EhVG3sYkw94

### Expected behavior

It should sound more like the second video

### Steps to reproduce the behaviour

Go to Zsnes
Load a game
And listen to the sound. Affects all games

**Comment by grapeli** [comment]
You need to be more specific about what's going on.
I compared the sound quality to dosbox-staging. It is basically the same. What's the bug in dosbox-x?

In the second video recording, the frequency does not exceed 6kHz, so the sampling bandwidth was 11025Hz. Low frequencies, bass are extremely strong. It sounds as if you covered the speaker with a pillow. How were these low frequencies boosted? In the Zsnes settings? With the use of a filter while ripping the video? In an unidentified DOS emulator?

**Comment by ualaa1** [comment]
What is going on is that the pitch on Dosbox-X when running SNES roms in Zsnes is wrong, so therefore it is offkey.
Yes, the sound quality for the second video is lower, but that is cause I used 86-Box running at 133 mhz. If I ran any faster 86-Box will lag. I had to lower the sound hz when running Zsnes in 86-box, otherwise 86-box will crash and freeze when running games.
The most important part is that even though 86-box (second video) has lower sound quality, it has the correct **pitch**
Dosbox-X has it wrong.

tl;dr
Pitch, not sound quality, is wrong
